python/test/unit/language/test_core_amd.py

```diff
@@ -1066,6 +1118,9 @@ def test_dot(M, N, K, num_warps, col_a, col_b, epilogue, allow_tf32, dtype, devi
     if capability[0] == 7:
         if (M, N, K, num_warps) == (128, 256, 32, 8):
             pytest.skip("shared memory out of resource")
+    if torch.version.hip is not None:
```

Comment:
Is this OOM skip check just a temporary thing?

IOW, are we sure that this is not fixable?